unlock-app/src/components/content/certification/CertificationDetails.tsx

```diff
@@ -249,7 +254,11 @@ export const CertificationDetails = ({
       } else {
         return (
           <span>
-            You are viewing the certificate issued to the recipient.{' '}
+            You are viewing the certificate issued to{' '}
```

Comment:
Actually can we change this to `You are viewing a CERTIFICATE NAME issued by ISSUER for 0x....`. This will make things even clearer IMO.